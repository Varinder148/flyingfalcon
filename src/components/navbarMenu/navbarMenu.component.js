import React from "react";
import { connect } from "react-redux";
import { resetFullGame } from "../../redux/player/player.action";
import "./navbarMenu.style.scss";

const NavbarMenu = ({ resetFullGame }) => {
  return (
    <ul className="navbar">
      <li onClick={resetFullGame}>Reset</li>
    </ul>
  );
};

const mapDispatchToProps = (dispatch) => ({
  resetFullGame: () => dispatch(resetFullGame()),
});

export default connect(null, mapDispatchToProps)(NavbarMenu);
