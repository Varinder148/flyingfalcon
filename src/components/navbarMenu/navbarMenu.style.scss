.navbar{
    display: inline-block;
    width:5rem;
    list-style: none;
    text-align: center;
    li:hover{
        padding:1rem;
        cursor: pointer;
        color:white;
        background: chocolate;
        border:1px solid chocolate;
        border-radius: 100%;
        
    }
}

