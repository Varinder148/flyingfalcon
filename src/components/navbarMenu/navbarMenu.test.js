import { shallow } from "enzyme";
import { NavbarMenu } from "./navbarMenu.component";

describe("Navbar tests", () => {
  it("checks if navbar is displayed", () => {
    const wrapper = shallow(<NavbarMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
