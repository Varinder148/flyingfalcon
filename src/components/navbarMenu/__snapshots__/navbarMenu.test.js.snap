// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar tests checks if navbar is displayed 1`] = `ShallowWrapper {}`;
